css/main.scss

```diff
@@ -108,13 +108,22 @@ p.divider {
 }
 
 // Sections and Layout =========================================================
+body {
+  margin:0;
+  position:relative; 
+ 
+}
 
 header {
 	.pagetitle {padding-top:30px;}
 	p {margin-bottom:0px;}
 }
 
-.content {padding:30px 0px;}
+.content {
+  padding:30px 0px;
+  padding-bottom: 15vh;
+
```

Comment:
nit) why the extra space here?

Reply:
so that the content at the very bottom of the page isn't hidden by the footer (so you can still scroll to it) - maybe there's a better way to do this